config-dev.js

```diff
@@ -38,3 +38,5 @@ global.LDAP_USER = "foo";
 global.LDAP_PASSWORD = "bar";
 global.LDAP_GROUPS = ["cn=123,ou=groups,dc=w3,dc=org", "cn=456,ou=groups,dc=w3,dc=org", "cn=32113,ou=groups,dc=w3,dc=org"];
 global.SKIP_VALIDATION = true
+
+global.TRAVIS_IP = ["35.224.112.202", "34.122.208.80", "34.74.253.255", "35.192.136.167", "104.154.113.151", "35.202.145.110", "35.192.85.2", "35.222.7.205", "207.254.16.36", "34.66.50.208", "35.192.187.174", "207.254.16.38", "34.73.65.1", "34.66.25.221", "35.192.10.37", "104.196.53.161", "35.202.245.105", "34.74.79.111", "35.192.91.101", "34.73.66.97", "35.237.56.208", "104.198.131.58", "35.227.97.188", "35.229.115.143", "104.196.57.92", "104.154.120.187", "34.122.173.211", "104.197.122.201", "35.231.58.0", "35.188.73.34", "34.74.91.53", "34.68.144.114", "35.193.184.18", "35.227.58.83", "35.237.8.208", "34.66.200.49", "35.196.82.30", "34.74.16.120", "34.73.34.132", "35.196.72.151", "35.193.7.13", "35.184.226.236", "35.185.97.135", "35.196.158.85", "207.254.16.35", "104.196.213.122", "35.188.15.155", "207.254.16.39", "104.154.182.187", "34.66.178.120", "207.254.16.37", "35.196.99.99", "35.237.212.185", "35.193.14.140", "35.188.1.99"]
```

Comment:
Similarly to what you are doing to automatically retrieve the GitHub servers IP addresses, you could do the same for the Travis CI servers.

From: https://docs.travis-ci.com/user/ip-addresses/

> Note that these ranges can change in the future, and the best way to keep an updated safelist is to use the current A records for the NAT hostname shown above, such as by using dig:
>
> `dig +short nat.travisci.net | sort`
>
> or using a service like dnsjson.com to interact with JSON:
>
> `curl -s https://dnsjson.com/nat.travisci.net/A.json | jq '.results.records|sort'`

The first command that relies on the DNS A records is the fastest one and should also be quite reliable removing the need to store those addresses locally (having a separate cron does not seem mandatory, this could be done real time in the code).

Side note: that parameter should not be part of the config-dev.js file

Reply:
addressed in https://github.com/w3c/echidna/pull/810/commits/19028c0674c911458647254698beed59ea8c6fd2